ncm-pam/src/main/pan/components/pam/schema.pan

```diff
@@ -38,9 +38,10 @@ type component_listfile_acl = {
 };
 
 type component_pam_module_stack = {
-	"control" : string with match(SELF, "requisite|required|optional|sufficient")
+	"control" : string with match(SELF, 'requisite|required|optional|sufficient|^\[(.*=.* ?)+\]$')
```

Comment:
why is the match not `^(requisite|required|optional|sufficient|\[(.*=.* ?)+\])$`
also `.* ?` makes no sense, you probably meant `[^=]+=[^=]+(\s+[^=]+=[^=]+)*`